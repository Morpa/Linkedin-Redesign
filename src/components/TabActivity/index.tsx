import * as S from './styles'

const TabActivity = () => (
  <S.Wrapper>
    <h1>TabActivity</h1>
  </S.Wrapper>
)

export default TabActivity
