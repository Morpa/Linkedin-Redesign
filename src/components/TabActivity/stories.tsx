import { Story, Meta } from '@storybook/react/types-6-0'
import TabActivity from '.'

export default {
  title: 'TabActivity',
  component: TabActivity
} as Meta

export const Default: Story = () => <TabActivity />
