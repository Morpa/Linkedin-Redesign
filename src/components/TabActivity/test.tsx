import { render, screen } from '@testing-library/react'

import TabActivity from '.'

describe('<TabActivity />', () => {
  it('should render the heading', () => {
    const { container } = render(<TabActivity />)

    expect(
      screen.getByRole('heading', { name: /TabActivity/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
