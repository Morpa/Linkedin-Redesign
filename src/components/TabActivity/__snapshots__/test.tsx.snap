// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabActivity /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    TabActivity
  </h1>
</main>
`
